lib/patients_page.dart

```diff
@@ -2,214 +2,12 @@ import 'package:flutter/material.dart';
 import 'package:hci_app/patient_detail_page.dart';
 
 class PatientsPage extends StatefulWidget {
+  const PatientsPage({Key key}) : super(key: key);
   @override
   _PatientsPageState createState() => _PatientsPageState();
 }
 
 class _PatientsPageState extends State<PatientsPage> {
-  List<Patient> patients = [
```

Comment:
@fischja Took the list out of the class so i could use the same patients in the activities